src/server.c

```diff
@@ -2870,12 +2870,20 @@ void resetServerStats(void) {
     server.blocked_last_cron = 0;
 }
 
+/* Make the thread killable at any time, so that kill threads functions
+ * ca work reliably. */
```

Comment:
```suggestion
/* Make the thread killable at any time, so that kill threads functions
 * can work reliably. See pthread_cancel used by the fast memory test used
 * by the crash report. */
```